api/controllers/app.js

```diff
@@ -105,14 +106,25 @@ router.get('/query', common.jwt({credentialsRequired: false}), (req, res, next)=
                 {projects: null}, //if projects is set to null, it's available to everyoone
                 {projects: {$exists: false}}, //if projects not set, it's availableo to everyone
             ]
-        })
+        };
+        if(req.query.find) findQuery = {$and: [findQuery, JSON.parse(req.query.find)]};
+        if(req.query.incompatible === 'true') {
+            // When incompatible flag is true, remove the datatype filter
+            if(findQuery.$and) {
+                findQuery.$and = findQuery.$and.filter(query => !query["inputs.datatype"]);
+            }
+        }
```

Comment:
The issue with this is that the `limit` may cut out the compatible apps, only displaying then the incompatible ones.
There are two ways of going about this:
1. Doing two separate queries, one for compatible and one for incompatible, in which the # of apps from the first reduces the number of apps allowed (limit) in the second
2. Create an aggregation to get all the results and already sort by # of incompatible inputs

Reply:
We could go with #2 and get rid of some processing on the ui.

Reply:
Just to be clear, #1 and #2 are in the backend